examples/undocumented/python/graphical/lda.py

```diff
@@ -1,32 +1,31 @@
-from pylab import figure,pcolor,scatter,contour,colorbar,show,subplot,plot,connect
+import matplotlib.pyplot as plt
 from shogun import *
 import util
 
-util.set_title('LDA')
```

Comment:
for the record. next time please make a smaller PR title. This one is way too epic ;)